staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -197,16 +197,17 @@ type priorityLevelState struct {
 	pl *flowcontrol.PriorityLevelConfiguration
 
 	// qsCompleter holds the QueueSetCompleter derived from `config`
-	// and `queues` if config is not exempt, nil otherwise.
+	// and `queues`.
+	// Formerly: non-nil if config is not exempt, nil otherwise.
```

Comment:
I wouldn't add comments about what it formerly was - it's overcomplicating it imho.

Reply:
I left some comments about "formerly" and `false &&` as breadcrumbs while developing/debugging this PR.  They will be removed.